### PR TITLE
🐛 fix: Improved styling of highlight code blocks

### LINKED ISSUE #62: Code blocks have too much spacing between lines
# Summary

Hugo's [`highlight` syntax highlighter](https://gohugo.io/functions/highlight/) has too much whitespace between lines, and lines of code do not match with line numbers.

# How to reproduce?

1. Create a block of code inside a content file using `{{< highlight >}}`
2. Run `hugo serve`
3. Go to the page with the block of code

# Expected behavior

A code block appears with no spacing between lines. Each line of code matches up to the line numbering in the highlighter sidebar.

# Actual behavior

There are two new lines between each individual line of code. The lines of code do not match up to the lune numbering in the highlighter sidebar.

**Screenshots**:

Screenshot of a code block with too much spacing between the lines. Lines of code do not match up with numbered lines.

# Other details

An example of how we currently use the [`highlight` function](https://gohugo.io/functions/highlight/) in our site (see it [_here_](https://github.com/unicef/inventory/blob/main/content/meta/inventory/maintainers-guide.en.adoc#example-git-in-command-line)):

```sh
{{< highlight bash "linenos=table,hl_lines=8 15-17,linenostart=1" >}}
git submodule update --remote --rebase
cd themes/inventory/
git fetch
git checkout <latest commit hash>
cd ../../
git add themes/inventory
git commit --signoff
{{< / highlight >}}
```

### PR DESCRIPTION
**Pull request by BeeBombshell**
Fixes #62

Changes:

This PR resolves the styling issue previously faced while using the `highlight` function to create code blocks.

Screenshot of the change:

image

### CODE REVIEW

**Review by jwflory — COMMENTED** [review]
Hey @BeeBombshell, thanks for opening the Pull Request!

I checked this in on the Inventory theme example site and the Open Source Inventory. In both cases, I did not get the expected results.

In the Inventory theme example site, the line numbers match, but there is still a lot of space between each line:

Screenshot of Code and Syntax Highlighting section, which shows the highlighted sections aligned, but there are still newlines between each line.

In the Open Source Inventory site, the same effect (see [my branch](https://github.com/unicef/inventory/tree/change/highlight-fix-newlines)):

Screenshot from Maitainer's Guide, which shows the highlighted sections aligned, but there are still newlines between each line.

Could you narrow down the root cause for the whitespace showing up? Then this will satisfy #62.

**Comment by BeeBombshell** [comment]
Hi @jwflory, as I mentioned earlier, there is no in-built `highlight` parameter to change the line spacing of the code... However, I made some changes in the `style.css` file which seem to fix the issue, and the outcome is as follows:

css changes

**Review by jwflory — CHANGES_REQUESTED** [review]
Thanks @BeeBombshell, the fix looks good 👍🏻 One request. Could you add a comment above the CSS addition with a link to the original issue, #62? For these CSS changes, it helps to add a short informational comment on why we are making a change the way we are.

Once the comment is in place, I am ready to merge this one 👍🏻

File: assets/css/style.css [hunk 1]
```diff
@@ -718,6 +718,10 @@ textarea.form-control {
     font-size: 14px;
 }
```
Comment (BeeBombshell) [inline assets/css/style.css:720]: With reference to #62, added styling to the `highlight` function for improved UI of code blocks.

**Review by jwflory — APPROVED** [review]
Looks good, thanks @BeeBombshell! ✅ Merging this one in. 🌊

**Pull request merged**